terraform/modules/self-service/lb.tf

```diff
@@ -66,3 +66,22 @@ resource "aws_lb_listener" "cluster_https" {
     target_group_arn = "${aws_lb_target_group.task.arn}"
   }
 }
+
+resource "aws_lb_listener_rule" "disable_health_check_from_public" {
+  listener_arn = "${aws_lb_listener.cluster_https.arn}"
```

Comment:
don't need to interpolate any more

Reply:
oh yeah, everything else is still interpolated, so will do a wider clean-up exercise later